docs/docs/06_plugins/01_available-plugins/01-transactional/05-typeorm-adapter.md

```diff
@@ -0,0 +1,50 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# TypeOrm adapter
+
+## Installation
+
+<Tabs>
+<TabItem value="npm" label="npm" default>
+
+```bash
+npm install @nestjs-cls/transactional-adapter-typeorm
+```
+
+</TabItem>
+<TabItem value="yarn" label="yarn">
+
+```bash
+yarn add @nestjs-cls/transactional-adapter-typeorm
+```
+
+</TabItem>
+<TabItem value="pnpm" label="pnpm">
+
+```bash
+pnpm add @nestjs-cls/transactional-adapter-typeorm
+```
+
+</TabItem>
+</Tabs>
+
+## Registration
+
+```ts
+
+ClsModule.forRoot({
+    plugins: [
+        new ClsPluginTransactional({
+            imports: [
+              // module in which the database instance is provided
+              TypeOrmModule
+            ],
+            adapter: new TransactionalAdapterTypeOrm({
+                // the injection token of the database instance
+                dbInstanceToken: DB,
```

Comment:
```suggestion
                dataSourceToken: DataSource,
```

When using `@nestjs/typeorm`, the token will have to be `dataSourceToken: getDataSourceToken()` (I'll update the docs myself after merge, so you can leave it)